cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -145,9 +145,34 @@
         border-bottom: 1px solid @horizontal-rule;
         margin-bottom: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
     }
+
+    &__email {
+        .respond-to-min(@bp-sm-min, {
+            box-sizing: border-box;
+            width: 50%;
+            padding-left: unit( @grid_gutter-width / @base-font-size-px, em );
+            float: right;
+        });
+            
+        .o-email-signup {
+            padding: unit( @grid_gutter-width / @base-font-size-px, em );
+            border: 1px solid @block__border-top;
+            margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+            background-color: @block__bg;
+
+            .respond-to-range(@bp-sm-min, @bp-sm-max {
+                width: 100%;
+            });
+
+            .a-btn {
+                width: auto !important;
```

Comment:
why do you we need to use important here?

Reply:
Looks like it isn't necessary -- removed in https://github.com/cfpb/cfgov-refresh/pull/4101/commits/af4e72470bbcff1d8fc444d93342d027b8824dd7